src/code/sys_matrix.c

```diff
@@ -1,25 +1,25 @@
 #include <ultra64.h>
 #include <global.h>
 
-void Matrix_Init(GameState* state) {
+void SysMatrix_StateAlloc(GameState* state) {
```

Comment:
who renamed these to begin with? i know it was said that there was an inconsistency in the name in functions.h, but the names that were here before seem far better anyways, and i'm pretty sure the SysMatrix_ prefix names were actually the old names, which have now been restored in this pr

Reply:
I renamed them to this because I remember people talking about renaming them to `SysMatrix_`, and those are declared like that in `functions.h`.
If most people prefer to name them back to `Matrix_` I can do it.